src/core/jit/exec_const.rs

```diff
@@ -75,6 +75,13 @@ impl ConstValueExecutor {
         let store = exe.store().await;
         let synth = Synth::new(&plan, store, vars);
 
+        if let Some(_auth_n) = &plan.auth_n {
+            if let Err(err) = req_ctx.auth_ctx.validate(req_ctx).await.to_result() {
+                return Response::default()
+                    .with_errors(vec![Positioned::new(Error::from(err), Pos::default())]);
+            }
+        }
```

Comment:
in ``let store = exe.store().await;` we execute the actual plan, so we would like to do the auth before that

Reply:
thanks